frontend/src/components/details/NavbarComponent.jsx

```diff
@@ -8,20 +8,20 @@ import slugs from 'resources/slugs';
  * @author Parham Gandomkar, Irem Toroslu, Julian Oelhaf
  */
 
-const PanelComponent = (props) => {
+const NavbarComponent = (props) => {
     return (
         <>
             <div className='navbar' vertical='center' horizontal='space-between'>
-                <b>{props.scenarioName}</b>
-                <button href={slugs.details}>
+                <b className='PanelHeaderTitle'>{props.scenarioName}</b>
+                <button href={slugs.details} className='PanelHeaderTitle'>
                     <i className='fa fa-fw fa-heart' /> Favorites
                 </button>
-                <button href={slugs.details} onClick={props.onCompareClick}>
+                <button href={slugs.details} onClick={props.onCompareClick} className='PanelHeaderTitle'>
                     <i className='fa fa-fw fa-plus-circle' /> Add
                 </button>
             </div>
         </>
     );
 };
```

Comment:
@IremToroslu 

There is also a way, to change these attributes depending on the component they are in, but using their actually tag name

@ScoutAtlas Correct me if I'm wrong but in .css one could do:
```css
/* Style the navigation menu */
.navbar {
    width: 100%;
    background-color: #466e78;
    overflow: auto;
}

/* Navigation links */
.navbar button {
    float: left;
    padding: 12px;
    color: white;
    background-color: #466e78;
    text-decoration: none;
    font-size: 17px;
    width: 20%; /* Four equal-width links. If you have two links, use 50%, and 33.33% for three links, etc.. */
    text-align: right; /* If you want the text to be centered */
    border-style: none;
}

.navbar b {
    float: left;
    padding: 12px;
    color: white;
    text-decoration: none;
    font-size: 17px;
    width: 60%; /* Four equal-width links. If you have two links, use 50%, and 33.33% for three links, etc.. */
    text-align: left; /* If you want the text to be centered */
}
```

please kindly take a look at the [navbar.css](https://github.com/amosproj/amos-ss2021-carbon-footprint/blob/68f9010b3dc820771143c71132507bc91e560bcb/frontend/src/components/details/navbar.css) file :)

Reply:
I did and you are right 😄 @ScoutAtlas but I guess I also changed these as well. I will check them out. Thanks for the review 😄 @Waldleufer @ScoutAtlas